astropy/cosmology/io/mapping.py

```diff
@@ -148,19 +157,58 @@ def to_mapping(cosmology, *args, cls=dict):
          'Tcmb0': <Quantity 2.7255 K>, 'Neff': 3.046,
          'm_nu': <Quantity [0.  , 0.  , 0.06] eV>, 'Ob0': 0.04897,
          'meta': ...
+
+    The dictionary type may be changed with the ``cls`` keyword argument:
+
+        >>> from collections import OrderedDict
+        >>> Planck18.to_format('mapping', cls=OrderedDict)
+        OrderedDict([('cosmology', <class 'astropy.cosmology.flrw.FlatLambdaCDM'>),
+          ('name', 'Planck18'), ('H0', <Quantity 67.66 km / (Mpc s)>),
+          ('Om0', 0.30966), ('Tcmb0', <Quantity 2.7255 K>), ('Neff', 3.046),
+          ('m_nu', <Quantity [0.  , 0.  , 0.06] eV>), ('Ob0', 0.04897),
+          ('meta', ...
+
+    Sometimes it is more useful to have the name of the cosmology class, not
+    the object itself. The keyword argument ``cosmology_as_str`` may be used:
+
+        >>> Planck18.to_format('mapping', cosmology_as_str=True)
+        {'cosmology': 'FlatLambdaCDM', ...
+
+    The metadata is normally included as a nested mapping. To move the metadata
+    into the main mapping, use the keyword argument ``move_from_meta``. This
+    kwarg inverts ``move_to_meta`` in
+    ``Cosmology.to_format("mapping", move_to_meta=...)`` where extra items
+    are moved to the metadata (if the cosmology constructor does not have a
+    variable keyword-only argument -- ``**kwargs``).
+
+        >>> from astropy.cosmology import Planck18
+        >>> Planck18.to_format('mapping', move_from_meta=True)
+        {'cosmology': <class 'astropy.cosmology.flrw.FlatLambdaCDM'>,
+         'name': 'Planck18', 'Oc0': 0.2607, 'n': 0.9665, 'sigma8': 0.8102, ...
     """
     if not issubclass(cls, (dict, Mapping)):
         raise TypeError(f"'cls' must be a (sub)class of dict or Mapping, not {type(cls)}")
```

Comment:
Here, since `cls` is a class, `type` is misplaced, right?

Reply:
Yes. thanks! I'll fix it here.